beacon-chain/operations/attestations/kv/aggregated.go

```diff
@@ -111,10 +111,10 @@ func (p *AttCaches) SaveAggregatedAttestations(atts []*ethpb.Attestation) error
 
 // AggregatedAttestations returns the aggregated attestations in cache.
 func (p *AttCaches) AggregatedAttestations() []*ethpb.Attestation {
-	atts := make([]*ethpb.Attestation, 0)
-
 	p.aggregatedAttLock.RLock()
 	defer p.aggregatedAttLock.RUnlock()
+
+	atts := make([]*ethpb.Attestation, 0, len(p.aggregatedAtt))
```

Comment:
Same feedback here from @0xKiwi. I'll revert this one as well and revisit later (maybe).